src/utils/legacyLayerParamUtils.js

```diff
@@ -123,8 +130,17 @@ export function getLayersFromLegacyUrlParams(layersConfig, legacyLayersParam) {
                         }
                         // checking if a timestamp is defined for this layer
                         if (layerTimestamps.length > index && layerTimestamps[index]) {
-                            layer.timeConfig.updateCurrentTimeEntry(
+                            let year = layerTimestamps[index]
+                            if (
                                 layer.timeConfig.getTimeEntryForTimestamp(layerTimestamps[index])
+                                    .year
+                            ) {
+                                year = layer.timeConfig.getTimeEntryForTimestamp(
+                                    layerTimestamps[index]
+                                ).year
+                            }
+                            layer.timeConfig.updateCurrentTimeEntry(
+                                layer.timeConfig.getTimeEntryForYear(year)
                             )
```

Comment:
If I'm not mistaken this code is doing exactly the same as before but is more complex, before it was more consise.

Reply:
It seems that you're right